mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.java

```diff
@@ -738,30 +739,40 @@ private void launchVoiceRecognitionIfNeeded() {
     }
 
     private void openPendingNfcPageIfNeeded() {
-        if (mPendingNfcData == null || mConnection == null || !mStarted) {
+        if (mPendingNfcIntent == null || mPendingNfcIntent.getData() == null
```

Comment:
Not sure if `mPendingNfcIntent.getData()` can be null if `mPendingNfcIntent` isn't.